From 112d92118243fa1e430c021c6007ec4fcfcb930d Mon Sep 17 00:00:00 2001
Message-Id: <112d92118243fa1e430c021c6007ec4fcfcb930d.1387385974.git.minovotn@redhat.com>
In-Reply-To: <7cba796b16325e96d01e65bd265b4073ab103157.1387385974.git.minovotn@redhat.com>
References: <7cba796b16325e96d01e65bd265b4073ab103157.1387385974.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 17 Dec 2013 06:46:38 +0100
Subject: [PATCH 4/5] QMP: Document throttling parameters of
 __com.redhat_drive_add

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1387262799-10350-6-git-send-email-armbru@redhat.com>
Patchwork-id: 56296
O-Subject: [PATCH v2 5/6] QMP: Document throttling parameters of __com.redhat_drive_add
Bugzilla: 889051
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

From: Markus Armbruster <armbru@redhat.com>

The previous commit copied the code from RHEL-6, including the
throttling parameters added in RHEL-6 commit ea4348a "block: Allow IO
throttling fields in __com.redhat_drive_add".  That commit neglected
to update documentation, then in qemu-monitor.hx, now in
qmp-commands.hx.  Update it, so that qmp-commands.txt is complete.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 qmp-commands.hx | 6 ++++++
 1 file changed, 6 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qmp-commands.hx | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/qmp-commands.hx b/qmp-commands.hx
index 9e742d5..f6fae33 100644
--- a/qmp-commands.hx
+++ b/qmp-commands.hx
@@ -137,6 +137,12 @@ Arguments:
 - "serial": Drive serial number (json-string, optional)
 - "snapshot": Enable snapshot mode (json-bool, optional)
 - "copy-on-read": Enable copy-on-read mode (json-bool, optional)
+- "bps": total throughput limit in bytes per second is specified
+- "bps_rd": read throughput limit in bytes per second is specified
+- "bps_wr": write throughput limit in bytes per second is specified
+- "iops": total I/O operations per second is specified
+- "iops_rd": read I/O operations per second is specified
+- "iops_wr": write I/O operations per second is specified
 
 Example:
 
-- 
1.7.11.7

